src/divider/Divider.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { View, StyleSheet } from 'react-native';
+import colors from '../config/colors';
+
+let styles = {};
+
+const Divider = ({style}) => (
+  <View style={[styles.container, style && style]} />
+);
+
+Divider.propTypes = {
+  style: View.propTypes.style,
+};
+
+styles = StyleSheet.create({
+  container: {
+    height: 1,
+    backgroundColor: colors.grey5
+  }
+});
+
+export default Divider;
```

Comment:
Let's make sure we are exposing this component in `index.js`and also documenting it.